scripts/launch/demo.sh

```diff
@@ -81,22 +79,21 @@ while [[ $i -lt $panesCnt ]]; do
       fi
   fi
 
-  stake_distr=" --flat-distr \"($n, 100000)\" "
-  kademlia_dump_path="kademlia$i.dump"
-  static_peers=''
-
-  if [[ $STATIC_PEERS != "" ]]; then
-      static_peers=' --static-peers'
-  fi
+  stake_distr=" --rich-poor-distr \"($n,50000,6000000000,0.99)\" "
```

Comment:
Fair enough. I made this change because the old `--flat-distr` caused the light wallet to complain about not having enough money. We could reorganize this script to take the distribution as a parameter.